sources/dicom/large_image_source_dicom/girder_source.py

```diff
@@ -59,15 +59,14 @@ def _getFilesystemLargeImagePath(self):
 
     def _getDICOMwebLargeImagePath(self, assetstore):
         meta = assetstore[DICOMWEB_META_KEY]
-        file = Item().childFiles(self.item, limit=1)[0]
-        file_meta = file['dicomweb_meta']
```

Comment:
This PR is mostly backward-compatible with existing imports. However, we previously stored the study/series UIDs in the file (within the item) and it was in the `dicomweb_meta` key. Now, the study/series UIDs are stored in the item, and the key is `dicom_uids`.

Do we want to keep this PR backward-compatible with existing imports? If so, I can first check if `dicom_uids` is present in the item. If not, then I can look for `dicomweb_meta` in the file.

Reply:
We don't need to keep it backward-compatible.